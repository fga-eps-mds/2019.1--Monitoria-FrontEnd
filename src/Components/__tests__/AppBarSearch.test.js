import React from 'react';
import AppBarSearch from '../Search/AppBarSearch';
import { shallow } from 'enzyme';

describe('Testing AppBarSearch component', () => {
    it('Test if AppBarSearch renders correctly', () =>{
        const tree = shallow(<AppBarSearch/>);
        expect(tree).toMatchSnapshot();
    });
});
