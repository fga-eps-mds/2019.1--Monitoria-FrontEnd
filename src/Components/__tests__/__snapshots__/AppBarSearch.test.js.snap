// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AppBarSearch component Test if AppBarSearch renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(SearchAppBar) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "grow": "SearchAppBar-grow-2",
        "inputInput": "SearchAppBar-inputInput-7",
        "inputRoot": "SearchAppBar-inputRoot-6",
        "root": "SearchAppBar-root-1",
        "search": "SearchAppBar-search-4",
        "searchIcon": "SearchAppBar-searchIcon-5",
        "title": "SearchAppBar-title-3",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "grow": "SearchAppBar-grow-2",
          "inputInput": "SearchAppBar-inputInput-7",
          "inputRoot": "SearchAppBar-inputRoot-6",
          "root": "SearchAppBar-root-1",
          "search": "SearchAppBar-search-4",
          "searchIcon": "SearchAppBar-searchIcon-5",
          "title": "SearchAppBar-title-3",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
